environments/extractDiffs.toml

```diff
@@ -0,0 +1,125 @@
+        deployed = 10322821
+    [contract.OASIS_MATCHING_MARKET_ONE]
```

Comment:
Might be able to remove the Oasis + OSM contracts?

Reply:
Updated in c0947f8